platform/config/search-promoted-pages.json

```diff
@@ -0,0 +1,15 @@
+{
+	"default": {
+		"components": [
+			"/content/amp-dev/documentation/components/reference/amp-img.md",
+			"/content/amp-dev/documentation/components/reference/amp-bind.md",
+			"/content/amp-dev/documentation/components/reference/amp-form.md"
+		],
+		"articles": [
+			"/content/amp-dev/documentation/guides-and-tutorials/start/converting/index.md",
+			"/content/amp-dev/documentation/guides-and-tutorials/learn/common_attributes.md",
+			"/content/amp-dev/documentation/guides-and-tutorials/develop/media_iframes_3p/index.md"
```

Comment:
replace with https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/